coralproject/talk
# Refactor and reuse "Actions" in "Moderate" and "Community"

## What does this PR do?
- Refactor `SuspendUserDialog` and the `BanUserDialog` into Top-Level Containers and Components.
- Reuse same logic across `Moderation` and `Community` tab.
- Rename previous `SuspendUserDialog` and related components of  `Community` to `RejectUsernameDialog`.

## How do I test this PR?

- Actions dropbox must appear in the `Flagged User` view of  `Community`
- Verify that the Ban, Suspend and RejectUsername workflow functions as expected.
